src/conf.h

```diff
@@ -52,6 +52,9 @@
 #define DEFAULT_GATEWAY_IPRANGE "0.0.0.0/0"
 #define DEFAULT_GATEWAYNAME "NoDogSplash"
 #define DEFAULT_GATEWAYPORT 2050
+#define DEFAULT_FORWARDINGPORT "2051"
```

Comment:
Why not a number instead of a string?

Reply:
Forwardingport has to be a string for use in splash.html including incorporation into $metaredirect.
Conversely it has to be a number for iptables. Converting to number from string was easier.

Reply:
usually you would do
```
char *str;
// ..
safe_asprintf(&str, "%d", config->some_port);
tmpl_set_variable(&templor, "some_port", str);
// ..
free(str);
```